esphome/core/color.h

```diff
@@ -43,7 +43,9 @@ struct Color {
                                                             b((colorcode >> 0) & 0xFF),
                                                             w((colorcode >> 24) & 0xFF) {}
 
-  inline bool is_on() ALWAYS_INLINE { return this->raw_32 != 0; }
+  inline Color(const Color &other_color) : raw_32(other_color.raw_32) {}
```

Comment:
AFAIK the C++ convention for copy constructors is to call the original "other", but I might be mistaken here.
```suggestion
  inline Color(const Color &other) : raw_32(other.raw_32) {}
```